tests/general/SpaceConsistencyBearTest.py

```diff
@@ -70,3 +75,22 @@ def test_enforce_newline_at_eof(self):
                                "    print('funny')\n",
                                "    print('the result is not funny...')"],
                               force_linebreaks=False)
+
+    def test_trailing_blanklines(self):
+        self.section.append(Setting('use_spaces', 'true'))
+        self.section.append(Setting('allow_trailing_whitespace', 'false'))
+        self.section.append(Setting('enforce_newline_at_EOF', 'true'))
+        self.section.append(Setting('allow_trailing_blanklines', 'false'))
+
+        self.check_invalidity(self.uut,
+                              ['def funny_code():\n',
+                               "  print('Is it funny?')\n",
+                               "  print('Yeah, funny')\n",
+                               '  \n',
+                               '\n'],
+                              force_linebreaks=False)
+        self.check_validity(self.uut,
+                            ['def funny_code():\n',
+                             "  print('Is it funny?')\n",
+                             "  print('Yeah, funny')\n"],
+                            force_linebreaks=False)
```

Comment:
I want to see tests for
- an empty file
- a file with only a single blank line
- a file with only multiple blank lines
- a file with leading and trailing blank lines